lib/info-extras.js

```diff
@@ -160,7 +160,8 @@ exports.getRelatedVideos = (body) => {
           author_thumbnail: details.channelThumbnail.thumbnails[0].url,
           short_view_count_text: shortViewCount.split(' ')[0],
           view_count: viewCount.replace(',', ''),
-          length_seconds: Math.floor(parseTime.humanStr(details.lengthText.simpleText) / 1000)
+          length_seconds: Math.floor(parseTime.humanStr(details.lengthText.simpleText) / 1000),
+          video_thumbnail: details.thumbnail.thumbnails[0].url
```

Comment:
can you add a comma at the end of the line here too?

Reply:
That's what I would have done but I followed the previous line's format :)
I'll add it